business_requirement/tests/test_br.py

```diff
@@ -113,19 +113,21 @@ def test_create_sequance(self):
             'name': '/',
             'description': 'test',
         }
-        self.br.create(br_vals)
+        res = self.br.create(br_vals)
+        self.assertTrue(res.name)
```

Comment:
@victormartinelicocorp I wish to compare it but I don't know what will be the next sequence number when test case will run. I cannot hard code the condition as you gave in the example.
Let me know if there is any way to compare it.